src/components/11-button-groups/button-groups.config.yml

```diff
@@ -0,0 +1,16 @@
+label: Button groups
+status: wip
```

Comment:
Are these statuses actually being used?

Reply:
@thisisdano ☝️

Reply:
Not really — if they are useful to you, continue using them, but the status doesn't output to the website in any way

Reply:
Gotcha - I switch to ready, but good to know moving forward.